server/server.go

```diff
@@ -48,7 +48,7 @@ func ListenAndServe(ctx context.Context, httpsPort, httpPort int, handler http.H
 	}
 
 	logger := logrus.StandardLogger()
-	errorLog := log.New(logger.WriterLevel(logrus.DebugLevel), "", log.LstdFlags)
+	errorLog := log.New(logger.WriterLevel(logrus.GetLevel()), "", log.LstdFlags)
```

Comment:
I like the simplicity of this solution, but I'm a bit concerned that this will always produce logs where before we weren't ever producing logs. Does this just log panics, or does this log a lot more?